com.reprezen.swagedit/src/com/reprezen/swagedit/assist/SwaggerContentAssistProcessor.java

```diff
@@ -48,15 +51,36 @@
  * This class provides basic content assist based on keywords used by the
  * swagger schema.
  */
-public class SwaggerContentAssistProcessor extends TemplateCompletionProcessor implements IContentAssistProcessor {
+public class SwaggerContentAssistProcessor extends TemplateCompletionProcessor implements IContentAssistProcessor, ICompletionListener {
 
 	private final SwaggerProposalProvider proposalProvider = new SwaggerProposalProvider();
+	private final JsonReferenceProposalProvider referenceProposalProvider = new JsonReferenceProposalProvider();
+	private final ContentAssistant contentAssistant;
+
 	private String currentPath = null;
+	private int cyclePosition = 0;
+	private boolean isRefCompletion = false;
+
+	private String[] textMessages = new String[] {
+			"Press 'Ctrl+Space' to show all schemas in the project.",
+			"Press 'Ctrl+Space' to show all schemas in the workspace.",
+			"Press 'Ctrl+Space' to show schemas in the current file."
```

Comment:
This is one of the cases which you referred to as "little bugs to fix"
'Ctrl+Space' binding is harcoded here. We have a requirement 
> JDT actually does detect the assigned shortcut key, and displays that key combination correctly in the prompt at the bottom of the suggestion list. We should do the same thing.
See  https://github.com/RepreZen/SwagEdit/issues/115#issuecomment-231142916